src/pages/_document.tsx

```diff
@@ -19,6 +19,14 @@ class MyDocument extends Document<DocumentProps> {
       <Html lang="en">
         <Head>
           <script src={appConfig.analytics} async />
+          {/* Google Tag Manager */}
+          {/* eslint-disable-next-line @next/next/next-script-for-ga */}
```

Comment:
Next third party scripts is available in Next 14

Reply:
Nice. Do you think it makes sense to upgrade to Next 15 sometime soon?

Reply:
Yes, we should also migrate to the app router to improve load states.